tests/integration.spec.coffee

```diff
@@ -149,13 +151,15 @@ describe 'SDK Integration Tests', ->
 
 				it 'should be rejected with an error', ->
 					promise = resin.auth.getEmail()
-					m.chai.expect(promise).to.be.rejectedWith(errors.ResinNotLoggedIn)
+					m.chai.expect(promise).to.be.rejected
+						.and.eventually.have.property('code', 'ResinNotLoggedIn')
```

Comment:
It should be possible to use the `name` property instead of `code` as well, since `name` is set at the root by typed-error (although it's possible that minification could screw that up)